lizmap/modules/lizmap/lib/Project/Project.php

```diff
@@ -1258,22 +1260,64 @@ public function getLoginFilters($layers, $edition = false)
             // attribute to filter
             $attribute = $loginFilteredConfig->filterAttribute;
 
+            # Quoted attribute with double-quotes
+            $cnx = \jDb::getConnection();
+            $quotedField = $cnx->encloseName($attribute);
+
             // default no user connected
-            $filter = "\"{$attribute}\" = 'all'";
+            $filter = "$quotedField = 'all'";
 
             // A user is connected
             if ($this->appContext->userIsConnected()) {
+                // Get the user
                 $user = $this->appContext->getUserSession();
                 $login = $user->login;
+
+                # List of values for expression
+                $values = array();
                 if (property_exists($loginFilteredConfig, 'filterPrivate')
                     && $this->optionToBoolean($loginFilteredConfig->filterPrivate)
                 ) {
-                    $filter = "\"{$attribute}\" IN ( '".$login."' , 'all' )";
+                    # If filter is private use user_login
+                    $values[] = $login;
                 } else {
+                    # Else use user groups
                     $userGroups = $this->appContext->aclUserPublicGroupsId();
-                    $flatGroups = implode("' , '", $userGroups);
-                    $filter = "\"{$attribute}\" IN ( '".$flatGroups."' , 'all' )";
+                    $values = $userGroups;
                 }
+
+                # Add all to values
+                $values[] = 'all';
+                $allValuesFilters = array();
+
+                # For each value (group, all, login, etc.), create a filter
+                # combining all the possibility: equality & LIKE
+                foreach ($values as $value) {
+                    $valueFilters = array();
+                    # Quote the value with single quotes
+                    $quotedValue = $cnx->quote($value);
```

Comment:
You know that if you want, you can safely remove this code and rely on the plugin part if needed.
At each release of server plugin, I'm increasing the minimum version required to run the LWC application.

Reply:
I could, but I need it for WFS when using a SQL request instead of asking QGIS Server. A parameter could help though...

Reply:
Is-it not the purpose of `REQUEST=GetSubsetString` ?

Reply:
I added a note in the method. We should discuss this in near future, in another dedicated PR